packages/statemanager/test/proofStateManager.spec.ts

```diff
@@ -17,16 +17,17 @@ tape('ProofStateManager', (t) => {
     const code = hexStringToBytes('6000')
     const stateManager = new DefaultStateManager()
     await stateManager.checkpoint()
+    await stateManager.putAccount(address, new Account())
     await stateManager.putContractStorage(address, key, value)
     await stateManager.putContractCode(address, code)
     const account = await stateManager.getAccount(address)
```

Comment:
I think you can do this:
```suggestion
    const account = (await stateManager.getAccount(address))!
```

This lets typescript know that it will not return `undefined`.

Reply:
Not sure about the context here, is this solved, respectively what was the problem to be solved here?

Reply:
It's not a problem at all, it is more a bit of a code helper / stylish thing. There are a lot of `account!.<field-to-index>` below, but if you let TS know that the account exists (so is not undefined) then all those `account!` lines below can just be renamed to `account`.